Integrations & Tutorials: Revamp

ngrayluna opened the pull request:
## Description

Removes unsupported (i.e. [not on this HEX dashboard](https://app.hex.tech/8ecc814e-8a34-46e1-b465-0c38bd16e93e/app/WREUs-by-Models-Integration-02zq9Q6r3z28vP0EYdIObf/latest)), *low page visits (Google Analytics), and/or broken notebooks (e.g. RayTune).

*Keeps top 10 most visited integration pages.

### Integration section changes

  - Landing page now has card-based layout that highlights popular ML integrations (PyTorch Lightning, HuggingFace, Keras, YOLOv5)                     
  - Word-smithed "Add wandb to any library"                
  - ML Frameworks: Reduced from 13 pages to 6, keeps:
     -  Keras, PyTorch, Lightning, Ignite, TensorFlow, and XGBoost                                                                            
  - ML Libraries: Reduced from 14 pages to 7, keeps:
     - Accelerate, Diffusers, SimpleTransformers, HuggingFace, OpenAI Gym, PyTorch Geometric, and Stable Baselines 3                      
  - Cloud Platforms: 
     - Azure OpenAI, OpenAI Fine-tuning, and OpenAI API                                                                                                          
  - Removed: Workflow Orchestration group (moved Hydra to **Other**)                                                                                                                             
 
### Tutorial section changes
- Entirely removes "Tutorials" seciton
- Links original notebooks to appropriate product pages (i.e. removes redundant mdx files that were created from notebooks) pages. 


### Redirects
Integration docs (integration/*) were not removed and no redirect has been added (yet). This is on purpose. For now, if you know the URL (.e.g saved as a bookmark), you'll still be able to access the page. When these docs migrate to Fully Connected, we'll remove the docs from wandb/docs and add the appropriate redirect (from docs.wandb.ai -> FC blog).

ngrayluna committed:
first draft

ngrayluna committed:
Remove duplicates

ngrayluna committed:
Merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna committed:
Added some integrations back

ngrayluna committed:
update landing page descriptions

ngrayluna committed:
Removed non HEX integrations

ngrayluna committed:
Merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna committed:
merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna committed:
Removed commented out content

ngrayluna committed:
Added redirect

ngrayluna committed:
Merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna committed:
wip

ngrayluna committed:
removed tutorials/experiments.mdx

ngrayluna committed:
removed tables tut

ngrayluna committed:
removed sweeps tut

ngrayluna committed:
removed artifacts and workspaces mdx

ngrayluna committed:
Removed Keras mdx tut

ngrayluna committed:
Removed keras models and keras tables mdx tut

ngrayluna committed:
removed unused Torch in integration section, moved Torch tut instead

ngrayluna committed:
wip add W&B word smithing

ngrayluna committed:
Merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna committed:
word smithed page

ngrayluna committed:
Merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna committed:
fixed eng link on add wb

ngrayluna committed:
Added tut redirects

mdlinville commented on models/integrations/add-wandb-to-any-library.mdx:
Drive-by: Maybe a good opportunity to rename the page.

johndmulhausen reviewed (changes requested):
Please also dissolve the "Tutorials" section and disperse the remaining tutorials into their respective feature areas in the Guides

johndmulhausen commented:
One more reorg step to do with the remaining tutorials but this is several steps in the right direction. Way to go, @ngrayluna !

ngrayluna committed:
Merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna commented:
> Please also dissolve the "Tutorials" section and disperse the remaining tutorials into their respective feature areas in the Guides

@johndmulhausen  I moved, consolidated what was obvious to move, but what about these remaining ones?

* [3D brain tumor segmentation with MONAI](https://docs.wandb.ai/models/tutorials/monai_3d_segmentation)
* Launch ([Volcano](https://docs.wandb.ai/models/tutorials/volcano) and [Minikube](https://docs.wandb.ai/models/tutorials/minikube_gpu)) <- We removed Launch pages

These tutorials can but put in a product page, but they'd get lost (ie. not discoverable). We might as well remove them because folks won't find them, imo:
* [TensorFlow sweeps](https://docs.wandb.ai/models/tutorials/tensorflow_sweeps) -> Sweeps
* [XGBoost sweeps](https://docs.wandb.ai/models/tutorials/xgboost_sweeps) -> Sweeps
* [Weave and Models](https://docs.wandb.ai/models/tutorials/weave_models_registry) -> Registry
* [Eval models w/ Weave and W&B Tables](https://docs.wandb.ai/models/evaluate-models) -> Tables

My rationale with this V1 of Integration and Tutorial revamp was to keep it as is and then on. On the second pass verify, delineate which actually have working *W&B-specific code (i.e. a true integration in our source code) vs. showing how to use W&B with some random library (i.e. no W&B specific code).

*Methinks it's worth making a distinction between showing how to generally use W&B with any library vs code we wrote specifically for that library and that users should use.

Thoughts?

ngrayluna committed:
Merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna committed:
removed TF and XGBoost examples

ngrayluna committed:
remove from docs.json

ngrayluna committed:
Removed old Launch tutorials

ngrayluna committed:
removed monai

ngrayluna committed:
removed from docs.json

ngrayluna committed:
Merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna committed:
moved, renamed files

ngrayluna committed:
Removed models and weave tutorial

ngrayluna committed:
Merge branch 'main' into integrations_tutorials_section_revamp_demo

ngrayluna committed:
removed tuts dir

ngrayluna committed:
updated docs.json

ngrayluna committed:
updated docs.json to use hf transformmers

ngrayluna committed:
Update redirects
